docs/writing-stories/decorators.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Decorators'
+---
+
+A decorator is a way to wrap a story in extra “rendering” functionality. Many addons define decorators in order to augment your stories with extra rendering or gather details about how your story is rendered.
+
+When writing stories, decorators are typically used to wrap stories with extra markup or context mocking.
+
+### Wrap stories with extra markup
+
+Some components require a “harness” to render in a useful way. For instance if a component runs right up to its edges, you might want to space it inside Storybook. Use a decorator to add spacing for all stories of the component.
+
+<div style="background-color:#F8FAFC">
+TODO: per screenshot spreadsheet add Image of Component Story without margins
+</div>
+
+```js
+export default {
+  component: TextComponent,
+  decorators: [(Story) => <div style={{ margin: ‘3em’ }}><Story/></div>]
+}
+```
+
+<div style="background-color:#F8FAFC">
+TODO: per screenshot spreadsheet add Image of Component Story with margins
+</div>
+
+
+### “Context” for mocking
+
+Some libraries require components higher up in the component hierarchy to render properly. For example in Styled Components, a `ThemeProvider` is required if your components make use of themes. Add a single global decorator that add this context to to all stories in __preview.js__ :
```

Comment:
We should always spell out `.storybook/preview.js` and link to the docs (people won't know what it is, necessarily).

Reply:
Just updated the wording. The link is now present,